logging/tests/system.py

```diff
@@ -171,6 +171,24 @@ def test_log_text_with_timestamp(self):
         self.assertEqual(entries[0].payload, text_payload)
         self.assertEqual(entries[0].timestamp, now.replace(tzinfo=UTC))
 
+    def test_log_text_with_resource(self):
+        from google.cloud.logging.resource import Resource
```

Comment:
Can you just import this globally (system test style slightly different than unit test style, thanks for trying to match though)